apis/kueue/v1alpha1/tas_types.go

```diff
@@ -16,7 +16,33 @@ limitations under the License.
 
 package v1alpha1
 
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
 const (
+	// PodSetTopologyRequiredAnnotation indicates that a PodSet requires
+	// Topology Aware Scheduling, and requires scheduling all pods on nodes
+	// within the same topology domain corresponding to the topology level
+	// indicated by the annotation value (e.g. within a rack or within a block).
+	PodSetTopologyRequiredAnnotation = "kueue.x-k8s.io/podset-required-topology"
```

Comment:
Can we keep the naming consistent between `PodSetTopologyRequiredAnnotation` and `PodSetPreferredTopologyAnnotation`? Should the Topology be before or after Required/Preferred?

Reply:
Sure, done